PROJECT

```diff
@@ -1,6 +1,6 @@
 version: "2"
-domain: cybozu.com
-repo: github.com/cybozu-go/topolvm
+domain: topolvm.io
+repo: github.com/topolvm/topolvm
```

Comment:
This file should not be modified yet.
Change this file when we add `topolvm.io` CRDs.